documentation for separate app and vendor entries (#3439)
webpack/webpack.js.org: src/content/concepts/entry-points.md

### Separate App and Vendor Entries

__webpack.config.js__

```javascript
module.exports = {
  entry: {
    main: './src/app.js',
    vendor: './src/vendor.js'
  }
};
```

__webpack.prod.js__

```javascript
module.exports = {
  output: {

>     filename: [name].[contentHash].bundle.js

Reviewer: Please use valid JavaScript

  }
};
```

__webpack.dev.js__

```javascript
module.exports = {
  entry: {
    filename: [name].bundle.js
  }
};
```

__What does this do?__ We are telling webpack that we would like 2 separate entry points (like the above example).

> __Why?__ The utility of this is that you can import required libraries or files that aren't modified (ex bootstrap, jquery, images, etc) inside vendor.js and they will be bundled together into their own HTML. Content Hash remains the same, which allows the browser to cache separately reducing load time.

Reviewer 2: ```suggestion
__Why?__ With this you can import required libraries or files that aren't modified (ex bootstrap, jquery, images, etc) inside `vendor.js` and they will be bundled together into their own HTML. Content Hash remains the same, which allows the browser to cache separately thereby reducing load time.
```
